src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = getSessionFactory();
```

Comment:
You should not initialize `instance` with `getSessionFactory()` at the moment of class loading. It's better to perform this operation inside `getSessionFactory()` method. This will allow to handle exceptions properly and only when you need to use SessionFactory.